index.html

```diff
@@ -98,89 +99,79 @@ <h2>The Finance and Investment club of NIT Trichy.</h2>
             </div>
           </div>
 
           <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-md-0">
             <div class="icon-box">
               <div class="icon"><i class="bx bx-file"></i></div>
-              <h4><a href="">Sed ut perspiciatis</a></h4>
-              <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore</p>
+              <h4><a href="">Insight</a></h4>
+              <p>Every year, a distinguished alumni in the field of finance is invited to interact with the student community and put forward their opinions on various upcoming domains of finance.</p>
             </div>
           </div>
 
           <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0">
             <div class="icon-box">
               <div class="icon"><i class="bx bx-tachometer"></i></div>
-              <h4><a href="">Magni Dolores</a></h4>
-              <p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia</p>
+              <h4><a href="">The Talk</a></h4>
+              <p>Every year we invite a Finance Leader to Campus to share their journey to the students and give them exposure from an Alumni who trod a different path to become a pioneer in their field.</p>
             </div>
           </div>
 
           <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4">
             <div class="icon-box">
               <div class="icon"><i class="bx bx-world"></i></div>
-              <h4><a href="">Nemo Enim</a></h4>
-              <p>At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis</p>
+              <h4><a href="">Guest Lectures and Webinars</a></h4>
+              <p>ProfNITT takes pride in having a strong mentor-alumni network, who are willing to shed light on interesting and important happenings in the world of finance and take out their time to interacting with us either offline or online.</p>
             </div>
           </div>
 
           <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4">
             <div class="icon-box">
               <div class="icon"><i class="bx bx-slideshow"></i></div>
-              <h4><a href="">Dele cardo</a></h4>
-              <p>Quis consequatur saepe eligendi voluptatem consequatur dolor consequuntur</p>
-            </div>
-          </div>
-
-          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4">
-            <div class="icon-box">
-              <div class="icon"><i class="bx bx-arch"></i></div>
-              <h4><a href="">Divera don</a></h4>
-              <p>Modi nostrum vel laborum. Porro fugit error sit minus sapiente sit aspernatur</p>
+              <h4><a href="">Case Solving Competitions</a></h4>
+              <p>The club also conducts various case-solving competitions throughout the year, where students try solving real-life industry problems.</p>
             </div>
           </div>

```

Comment:
1. 80 characters per line limit.
2. Are the used relevant to the events under question?

Reply:
1. Ok
2. Yes, got it from the Orientation Briefing Video.

Reply:
Are the used icons*

Reply:
We have to design icons or find something relevant to the events in question.